bin/yaml/c.yaml

```diff
@@ -28,6 +28,15 @@ compilers:
         - "2.17"
         - "2.18"
         - "2.19"
+    llvmmos:
+      type: tarballs
+      compression: xz
+      dir: llvm-mos-{name}
+      check_exe: bin/clang --version
+      url: https://github.com/llvm-mos/llvm-mos-sdk/releases/latest/download/llvm-mos-linux.tar.xz
```

Comment:
Can we have a tagged version for this? "trunk" only makes sense if we build/install this nightly (which this change won't do).

We can either:
- specify a named version here (well, as `{name}` here, and then with an appropraite `target:` below
- OR move this to the nightly section, and use `always install` type semantics

For the former obviously we'll need to keep adding here as new ones turn up and also keep updating the main CE repo. And it seems like this is fast moving and unstable, so probably not that.

For the second option you can add:
```yaml
if: nightly
type: nightlytarballs
```
here instead (well, not add the `type`, change the `type`. Hopefully that's clear.

Reply:
Ah, thanks, I was wondering what exactly nightlytarballs did.

That seems like the way to go, at least until development on the compiler cools down somewhat. The compiler's already very usable, but it's still moving a bit too fast to cut a SemVer 1.0. When that time comes, that seems like it'd  also be a good time to switch to a versioned release here.